public/styles/reset.css

```diff
@@ -7,6 +7,7 @@
 a,
 a:link,
 a:visited {
+  max-width: 300px;
```

Comment:
This should not be part of a reset. Can we scope this more narrowly?

Reply:
Yes, moved into .button class.